samples/bullet-samples/01_basics.py

```diff
@@ -0,0 +1,181 @@
+#from panda3d.core import load_prc_file_data

```

Comment:
This file should have UNIX newlines.

Also, every file here should have a shebang:
```
#!/usr/bin/env python
```